Dockerfile

```diff
@@ -44,34 +118,86 @@ RUN AVAILABLE_TERRAFORM_VERSIONS="1.0.11 1.1.9 1.2.9 ${DEFAULT_TERRAFORM_VERSION
     done && \
     ln -s "/usr/local/bin/tf/versions/${DEFAULT_TERRAFORM_VERSION}/terraform" /usr/local/bin/terraform
 
-# renovate: datasource=github-releases depName=open-policy-agent/conftest
-ENV DEFAULT_CONFTEST_VERSION=0.38.0
 
-RUN AVAILABLE_CONFTEST_VERSIONS="${DEFAULT_CONFTEST_VERSION}" && \
-    case ${TARGETPLATFORM} in \
-        "linux/amd64") CONFTEST_ARCH=x86_64 ;; \
-        "linux/arm64") CONFTEST_ARCH=arm64 ;; \
-        # There is currently no compiled version of conftest for armv7
-        "linux/arm/v7") CONFTEST_ARCH=x86_64 ;; \
-    esac && \
-    for VERSION in ${AVAILABLE_CONFTEST_VERSIONS}; do \
-        curl -LOs https://github.com/open-policy-agent/conftest/releases/download/v${VERSION}/conftest_${VERSION}_Linux_${CONFTEST_ARCH}.tar.gz && \
-        curl -LOs https://github.com/open-policy-agent/conftest/releases/download/v${VERSION}/checksums.txt && \
-        sed -n "/conftest_${VERSION}_Linux_${CONFTEST_ARCH}.tar.gz/p" checksums.txt | sha256sum -c && \
-        mkdir -p /usr/local/bin/cft/versions/${VERSION} && \
-        tar -C /usr/local/bin/cft/versions/${VERSION} -xzf conftest_${VERSION}_Linux_${CONFTEST_ARCH}.tar.gz && \
-        ln -s /usr/local/bin/cft/versions/${VERSION}/conftest /usr/local/bin/conftest${VERSION} && \
-        rm conftest_${VERSION}_Linux_${CONFTEST_ARCH}.tar.gz && \
-        rm checksums.txt; \
-    done
+# Stage 2 - Alpine
+# Creating the individual distro builds using targets
+FROM alpine:${ALPINE_TAG} AS alpine
 
-RUN ln -s /usr/local/bin/cft/versions/${DEFAULT_CONFTEST_VERSION}/conftest /usr/local/bin/conftest
+# atlantis user for gosu and OpenShift compatibility
+RUN addgroup atlantis && \
+    adduser -S -G atlantis atlantis && \
+    adduser atlantis root && \
+    chown atlantis:root /home/atlantis/ && \
+    chmod g=u /home/atlantis/ && \
+    chmod g=u /etc/passwd
 
 # copy binary
 COPY --from=builder /app/atlantis /usr/local/bin/atlantis
+# copy terraform
+# TODO: figure out better way to copy versions over, maybe just track one version
+COPY --from=deps /usr/local/bin/terraform /usr/local/bin/terraform
+COPY --from=deps /usr/local/bin/terraform1.2.9 /usr/local/bin/terraform1.2.9
+COPY --from=deps /usr/local/bin/terraform1.1.9 /usr/local/bin/terraform1.1.9
+COPY --from=deps /usr/local/bin/terraform1.0.11 /usr/local/bin/terraform1.0.11
+# copy deps
+COPY --from=deps /usr/local/bin/conftest /usr/local/bin/conftest
+COPY --from=deps /bin/gosu /bin/gosu
+COPY --from=deps /usr/bin/git-lfs /usr/bin/git-lfs
+# copy docker entrypoint
+COPY docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
+
+# Install packages needed for running Atlantis.
+# We place this last as it will bust less docker layer caches when packages update
+RUN apk add --no-cache \
+        ca-certificates~=20220614 \
+        curl~=7.87 \
+        git~=2.38 \
```

Comment:
Does `git~=2.38` allow the latest git 2.39 to be installed without using the separate package repo?

Ref: https://github.com/runatlantis/atlantis/pull/2998

Reply:
I doesn't appear it does: https://jubianchi.github.io/semver-check/#/~%3D2.38/2.39.0

Looks like we might have to do `git~=2` which honestly is weird to me.

Reply:
Seems ok to me, as long as it picks up the latest git security patch. If you're not comfortable with ~2, let's raise the pin to ~2.39.